platform-common-date/src/main/java/com/softicar/platform/common/date/IllegalDateSpecificationException.java

```diff
@@ -0,0 +1,32 @@
+package com.softicar.platform.common.date;
+
+import com.softicar.platform.common.core.exceptions.SofticarUserException;
+import com.softicar.platform.common.string.Padding;
+
+/**
+ * This {@link Exception} is thrown if an illegal date was specified.
+ *
+ * @author Oliver Richers
+ */
+public class IllegalDateSpecificationException extends SofticarUserException {
+
+	public IllegalDateSpecificationException(int year, int month, int day) {
+
+		this(
```

Comment:
We should probably check this formatting stuff in `IllegalDateSpecificationExceptionTest` (yet to be created).

Reply:
ok, did so, please check